### PR TITLE
CI: run with Symfony:^6

### PR DESCRIPTION
**Pull request by keradus**
POC, looks like some deps are not yet ready

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@ jobs:
             job-description: 'with Sf ^5'
             execute-flex-with-symfony-version: '^5' # Explicit check for Sf 5.x compatibility
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
```
Comment (keradus) [inline .github/workflows/ci.yml:29]:
```suggestion
            php-version: '8.0'
```

**Comment by derrabus** [comment]
Enabling Symfony 6 would be awesome. I'd like to test some applications on Symfony 6 that require PHP CS Fixer in their dev dependencies. I have continued this PR in #6095.

**Comment by julienfalque** [comment]
> applications on Symfony 6 that require PHP CS Fixer in their dev dependencies

Can't PHP CS Fixer be extracted from those dependencies by e.g. using a separate composer.json file?

**Comment by derrabus** [comment]
Well sure it could, but it would mean to also change the CI pipeline and developer tooling. Not your problem, I know.

Asking an application that apparently is already compatible with Symfony 6 to indicate that compatibility seemed like the easier step to me. But I can also work with my fork for the time being. No pressure. 🙂

**Comment by keradus** [comment]
closing as replaced

**Pull request closed without merging**